test/units/testsuite-54.sh

```diff
@@ -200,6 +200,11 @@ elif [ -d /sys/firmware/qemu_fw_cfg/by_name ]; then
     [ "$(cat /tmp/sourcedfromcredential)" = "tmpfilessecret" ]
     [ "$(cat /etc/motd.d/50-provision.conf)" = "hello" ]
     [ "$(cat /etc/issue.d/50-provision.conf)" = "welcome" ]
+
+    # Verify that adding a unit and drop-in via credentials worked
+    systemctl start my-service
+    sleep 1
```

Comment:
the sleep here is a bit risky, some CIs are really slow - how about making the unit write to /tmp/, and loop on existence of the file with a timeout? that's the usual pattern for things like this in the tests